.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test.md
```

Comment:
- [ ] Well done adding the test.md file to the .gitignore, but there is an empty file in your repo, it could be better if you add the phrase "This file should be ignored by git". And correct the path in the .gitignore 👉🏽 `./test.md`